src/server/chromium/chromium.ts

```diff
@@ -22,17 +22,22 @@ import { kBrowserCloseMessageId } from './crConnection';
 import { rewriteErrorMessage } from '../../utils/stackTrace';
 import { BrowserType } from '../browserType';
 import { ConnectionTransport, ProtocolRequest } from '../transport';
-import type { BrowserDescriptor } from '../../utils/browserPaths';
+import * as browserPaths from '../../utils/browserPaths';
 import { CRDevTools } from './crDevTools';
 import { BrowserOptions, PlaywrightOptions } from '../browser';
 import * as types from '../types';
 import { isDebugMode } from '../../utils/utils';
 
 export class Chromium extends BrowserType {
   private _devtools: CRDevTools | undefined;
+  private _ffmpegPath: string | null;
 
-  constructor(packagePath: string, browser: BrowserDescriptor, playwrightOptions: PlaywrightOptions) {
+  constructor(packagePath: string, browser: browserPaths.BrowserDescriptor, ffmpeg: browserPaths.BrowserDescriptor, playwrightOptions: PlaywrightOptions) {
     super(packagePath, browser, playwrightOptions);
+
+    const browsersPath = browserPaths.browsersPath(packagePath);
```

Comment:
Let's put everything we need in PlaywrightOptions or somewhere close, and introduce a helper method that will retrieve all the paths on demand? I feel like we have too much juggling with browserPaths scattered across the codebase.